src/ApiDecider.php

```diff
@@ -20,6 +20,22 @@ class ApiDecider
     /** @var ApiHandlerInterface|null */
     private $globalPreflightHandler = null;
 
+    /** @var BaseHandler */
+    private $defaultHandler;
+
+    /** @var ApiAuthorizationInterface */
+    private $authorizationForDefaultHandler;
+
+    /**
+     * @param BaseHandler|null $defaultHandler Default handler that will be called on unknown api endpoint
+     * @param ApiAuthorizationInterface|null $authorizationForDefaultHandler Authorization for default handler
+     */
+    public function __construct(?BaseHandler $defaultHandler = null, ?ApiAuthorizationInterface $authorizationForDefaultHandler = null)
```

Comment:
This is BC break

Also if you have registered some handler / authorization as services, they will be auto injected here, if there will be more than one such services, you will get nette DI error -> you should create some new interfaces which will be used here

I would recommend to implement this feature in your own apps by extending ApiDecider